data/json/recipes/recipe_food.json

```
@@ -2144,7 +2144,7 @@
     "charges": 10,
     "time": "20 m",
     "batch_time_factors": [ 50, 4 ],
-    "autolearn": true,
+    "book_learn": [ [ "baking_book", 2 ] ],
```

```suggestion
    "book_learn": [ [ "family_cookbook", 1 ], [ "baking_book", 1 ], [ "cookbook", 2 ], [ "cookbook_italian", 3 ], [ "atomic_survival", 6 ] ],
```
Trying to reduce the size of family_cookbook, but I've yet to see a family cookbook without a bread recipe.
We should make sure bread recipe is widespread since it's so important.